common/utils/works.js

```diff
@@ -225,14 +225,26 @@ export function getLocationType(work: Work, locationType: string) {
 }
 
 type LocationType = 'PhysicalLocation' | 'DigitalLocation';
+type Location = {|
+  locationType: {
+    id: string,
+    label: string,
+    type: 'LocationType',
+  },
+  label: string,
+  type: LocationType,
+|};
 
-export function getItemLocationsOfType(work: Work, locationType: LocationType) {
+export function getItemLocationsOfType(
+  work: Work,
+  locationType: LocationType
+): Location[] {
   const locations =
     work.items &&
     work.items
       .map(item => {
         return item.locations.find(location => location.type === locationType);
       })
       .filter(Boolean);
-  return locations && locations.length > 0 ? locations : null;
+  return locations || [];
```

Comment:
This will always be an array as a `map`/`filter` never returns null.

Reply:
`work.items && work.items…` might return `false` though?

Reply:
Bah, my bad.